.github/workflows/ci-app-accessibility-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: CI App Accessibility Tests
+
+on:
+  workflow_call:
+    inputs:
+      service_endpoint_url:
+        required: true
+        type: string
+
+jobs:
+  build:
+    name: Accessibility
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./app
+
+    env:
+      NODE_VERSION: 20
+      LOCKFILE_PATH: ./app/package-lock.json
+      PACKAGE_MANAGER: npm
```

Comment:
this workflow should be reusable and not specific to any app, so don't put it in the app folder. some alternatives:

- put it in the root folder
- put it in a folder called `accessibility`

i lean towards putting it in an `accessibility` folder. that way we can put configs in there, and potentially have the ability to have app-specific configs like `accessibility/<APP_NAME>/`

Reply:
IIUC - put a `package.json` under `./accessibility/` ?

Reply:
yup